### PR TITLE
Added comments and made example URL's more realistic.

### PR DESCRIPTION
**Pull request by jeid64**
Comments make my life better.

Is the forge link now required to be Github always or can it link to other places?

### CODE REVIEW

File: people/2015/spring/fake_student.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
+blog: http://yourbloghere.wordpress.com/ # Link to the home page of your blog here.
+feed: http://en.blog.wordpress.com/feed/ # Link to the RSS feed of your blog here. If you are using Blogger, your RSS feed is "http://blogname.blogspot.com/feeds/posts/default?alt=rss"
 forges:
-  - http://fsf.org
-irc: Thunderdome
-name: Thunder Dome
-rit_dce: abc123
+  - http://github.com/yourusernamehere # Github is recommended for this course but there are others.
+irc: Thunderdome # Your IRC username that you will be using in the rit-foss channel. This should always be consistent.
+name: Thunder Dome # Your legal name.
```
Comment (decause) [inline people/2015/spring/fake_student.yaml]: Not a thing anymore

**Comment by decause** [comment]
@jeid64 be sure to pull, and then, remove the <code>name</code> field.

Fix and I'll merge :wrench:

**Comment by jeid64** [comment]
Removed name field. Ready to merge!

**Comment by decause** [comment]
@jeid64 I still like this idea, but worry about comments-at-the-end-of-the-line making each entry longer than 79 characters.

Perhaps you can put all of the comments in the top block, so that the formatting isn't so complicated by word-wrap? 

And I failed to answer your question the first time about forges! The answer is, yes, there needs to be at least one entry in the forges list that goes to your github repository, but folks can add all the other places they host code to (i.e. bitbucket, sourceforge, self-hosted, etc...)

File: people/2015/spring/fake_student.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Fake Student Sample YAML #YOLOSWAG
-blog: http://wordpress.com
-feed: http://en.blog.wordpress.com/feed/
+blog: http://yourbloghere.wordpress.com/ # Link to the home page of your blog here.
```
Comment (Nolski) [inline people/2015/spring/fake_student.yaml:2]: Aren't these links still wrong?

Comment (liam-middlebrook) [inline people/2015/spring/fake_student.yaml:2]: Why would they be wrong? This is the example YAML file.

**Comment by Nolski** [comment]
Riiiigghhtt. It's too late for this stuff. I thought this was a first flight submission.

**Comment by decause** [comment]
Perhaps we should add this content of fake_student.yaml into the FirstFlight assignment, regardless of how this pans out?

**Comment by decause** [comment]
:+1:

**Pull request merged**